### PR TITLE
add cses-1631 solution

### PR DESCRIPTION
**Pull request by 876pol**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/cses-1631.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Let $s$ be $\sum\limits_{i=1}^n t[i]$.
+
+Sort the array in ascending order. Kotivalo can read the books in the order $t[n], t[1], t[2], ..., t[n - 1]$, and Justiina can read them in the order $t[1], t[2], t[3], ..., t[n]$. 
+
+If $s - t[n] < t[n]$, Justiina would finish reading books $t[1], t[2], t[3], ..., t[n - 1]$ and would have to wait for Kotivalo to finish reading $t[n]$. Afterwards, Justiina can read $t[n]$ and Kotivalo can read $t[1], t[2], t[3], ..., t[n - 1]$. In total, this will take $2 \times t[n]$ units of time.
+
+If $s - t[n] >= t[n]$, Kotivalo would finish reading $t[n]$ before Justiina finishes reading $t[1], t[2], t[3], ..., t[n - 1]$, so Justiina would not have to wait for Kotivalo to finish reading $t[n]$. Thus, this will take $s$ units of time.
+
+Therefore, the answer is $\max(s, 2 \times t[n])$.
```
Comment (ryanchou-dev) [inline solutions/cses-1631.mdx:20]: texttt `t`

File: solutions/cses-1631.mdx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+using namespace std;
+
+using ll = long long;
+
+int main() {
+	ll n;
+	cin >> n;
+	vector<ll> t(n);
+	for (ll i = 0; i < n; i++) cin >> t[i];
```
Comment (ryanchou-dev) [inline solutions/cses-1631.mdx:39]: wrap in braces

File: solutions/cses-1631.mdx [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+	ll mx = *max_element(t.begin(), t.end());
+	ll sum = accumulate(t.begin(), t.end(), 0ll);
+	cout << max(sum, mx * 2) << "\n";
```
Comment (ryanchou-dev) [inline solutions/cses-1631.mdx:42]: space some of this out

**Review by ryanchou-dev — CHANGES_REQUESTED** [review]
happy first contribution 🎉

File: solutions/cses-1631.mdx (hunk 1 above)
Comment (876pol) [reply solutions/cses-1631.mdx]: when should texttt be used? im new to latex

File: solutions/cses-1631.mdx [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+#include <bits/stdc++.h>
+using namespace std;
+
+using ll = long long;
+
+int main() {
+	ll n;
+	cin >> n;
+
+	vector<ll> t(n);
```
Comment (SansPapyrus683) [inline solutions/cses-1631.mdx:39]: (optional) rename to `books`

File: solutions/cses-1631.mdx [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+	for (ll i = 0; i < n; i++) {
+		cin >> t[i];
+	}
+
+	ll mx = *max_element(t.begin(), t.end());
+	ll s = accumulate(t.begin(), t.end(), 0ll);
```
Comment (SansPapyrus683) [inline solutions/cses-1631.mdx:45]: (optional) rename to `longest_time` and `total_time` respectively

File: solutions/cses-1631.mdx [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+	cout << max(s, mx * 2) << "\n";
+}
+```
+</CPPSection>
+</LanguageSection>
```
Comment (SansPapyrus683) [inline solutions/cses-1631.mdx:51]:
```py
n = int(input())
books = [int(b) for b in input().split()]

longest_time = max(books)
total_time = sum(books)

print(max(total_time, longest_time * 2))
```
if you could add my python solution that's be great

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

File: solutions/cses-1631.mdx (hunk 6 above)
Comment (876pol) [reply solutions/cses-1631.mdx:51]: what's your name? (so i can add you to the list of authors)

Comment (SansPapyrus683) [reply solutions/cses-1631.mdx:51]: nah u don't have to

**Review by SansPapyrus683 — APPROVED** [review]

**Review by ryanchou-dev — APPROVED** [review]
⭐

**Pull request merged**